lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -55,6 +56,7 @@ export abstract class PerformanceClient implements IPerformanceClient {
         this.callbacks = new Map();
         this.eventsByCorrelationId = new Map();
         this.measurementsById = new Map();
+        this.httpVer = "";
```

Comment:
```suggestion
        this.httpVer = "";
```

Should this be an empty string constant instead?